eng/Versions.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>3.1.0</VersionPrefix>
-    <PreReleaseVersionLabel>preview4</PreReleaseVersionLabel>
+    <VersionPrefix>3.2.0</VersionPrefix>
+    <PreReleaseVersionLabel>preview1</PreReleaseVersionLabel>
```

Comment:
Now that you have moved to a new minor version, can you update the version iteration label so that the pre-release label is 'preview.1'?

See: https://github.com/dotnet/windowsdesktop/pull/104/files

Reply:
ASPNetCore was supposed to change too for .NET 5, but hasn't yet. There is an issue open.

Reply:
When will blazor move to targeting net5 (and flowing to aspnetcore master?). That might be the right time to do this for this repo.

Reply:
I believe they plan to never do that - @javiercn @pranavkm @mkArtakMSFT please confirm

Reply:
> I believe they plan to never do that - @javiercn @pranavkm @mkArtakMSFT please confirm

You're referring to blazor move to targeting net5 w.r.t "never do that?"

Reply:
Yes

Reply:
Blazor is definitely part of .NET 5. I'm not sure what is being suggested here.

I agree that we could leave the package naming as-is in this branch, and only make the change in the `master` branch for 5.0.

Reply:
This is the `master` branch, but since the only active work in this repo is for `blazor-wasm` which is a .Net 3 product, this branch flows into the `blazor-wasm` branch of AspNetCore, not the `master` branch of AspNetCore. I was under the impression that Blazor was eventually going to be folded into the SDK for 5.0 rather than having its own repo, but maybe I misunderstood the plan

Reply:
> Blazor was eventually going to be folded into the SDK for 5.0

Blazor the repo, not Blazor the product.  I would expect in the 5.0 timeframe, we wouldn't need to do this re-packaging and the artifacts we depend on would come out of `dotnet/runtime`.

Reply:
@Pilchie is right. This package is only for the blazor-wasm timeframe and we don’t expect to ship in the 5.0 timeframe, so this is the right branding.

At some point in the future this will change.